boards/rapidsilicon/virgo_tc_eval_soc_proto/board.yml

```
@@ -0,0 +1,5 @@
+board:
+  name: virgo_tc_eval_soc_proto
```

```suggestion
  name: virgo_eval_proto
```
Let me know if you disagree

---

gave it virgo_proto